client/src/App.tsx

```diff
@@ -1,20 +1,37 @@
-import {observer} from "mobx-react-lite";
-import {Error404} from "pages/Error404/Error404";
-import {BrowserRouter as Router, Route, Routes} from 'react-router-dom';
+import { observer } from "mobx-react-lite";
+import { Error404 } from "pages/Error404/Error404";
+import { BrowserRouter as Router, Route, Routes } from 'react-router-dom';
 import GamePage from 'pages/GamePage';
-import {NewGamePage} from 'pages/NewGamePage';
-import {Rooms} from 'components/Rooms/Rooms';
+import { Rooms } from 'components/Rooms/Rooms';
+import { Keyboard } from "./pages/Keyboard";
+import { CreateGamePage } from "./pages/CreateGamePage";
+import React from "react";
+import appStore from "./stores/appStore";
+import { EnterNamePage } from "./pages/EnterNamePage";
```

Comment:
Don't forget to export these components `as default`.
```suggestion
import React from "react";
import appStore from "./stores/appStore";
const Rooms = React.lazy(() => import("components/Rooms/Rooms"))
const Keyboard = React.lazy(() => import("./pages/Keyboard"))
const CreateGamePage = React.lazy(() => import("./pages/CreateGamePage"))
const EnterNamePage = React.lazy(() => import("./pages/EnterNamePage"))
```

Reply:
After all, can you add the **dead heat** case? Because now in case of the same amount of points we anyway have a winner (the first player). 

The screenshot with proof:
<img width="1280" alt="Screenshot 2022-12-22 at 20 59 54" src="https://user-images.githubusercontent.com/38957112/209206295-ae650116-87ae-4eba-a648-7c2d20d69460.png">